pkgs/development/python-modules/pgpy/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, isPy3k, fetchFromGitHub, buildPythonPackage
+, six, enum34, pyasn1, cryptography, singledispatch
+, flake8, pytest, pytestcov }:
+
+buildPythonPackage rec {
+  pname = "pgpy";
+  version = "0.4.3";
+
+  # Pypi lacks tests/
+  src = fetchFromGitHub {
+    owner = "SecurityInnovation";
+    repo = "PGPy";
+    rev = version;
+    sha256 = "0alp84bnfklzyybr2prw7y7z1vvrwnhsvn5g2wqhvxdzf3x43w4f";
+  };
+
+  propagatedBuildInputs = [
+    six
+    pyasn1
+    cryptography
+    singledispatch
+  ] ++ lib.optional (!isPy3k) enum34;
+
+  # Requires gpg@1.8.0, pep8-naming, and pytest-ordering
+  doCheck = false;
+  checkInputs = [
+    flake8
+    pytest
+    pytestcov
+  ];
+
+  # Exclude test in root directory from 5 years ago, test_load_asc_bench.py
+  # Has progressbar dependency that doesn't look like even supports Python 3.x
```

Comment:
There are multiple progressbar successors in our `python-packages.nix`, some of which support Python 2 and 3.

Reply:
That error should be reported upstream if it has not been fixed yet.

To fix the progressbar2 build, I have pushed 667230441802bcbf278eaf928df28643fbb91ee2 to master.

Reply:
https://github.com/SecurityInnovation/PGPy/issues/250